Packs/Devo/Integrations/Devo_v2/Devo_v2.py

```diff
@@ -65,171 +62,224 @@
+            oauth_token=READER_OAUTH_TOKEN,
+            end_point=READER_ENDPOINT,
+            verify=not ALLOW_INSECURE,
+        ).query(
+            HEALTHCHECK_QUERY,
+            start=int(time.time() - 1),
+            stop=int(time.time()),
+            output="dict",
+        )
+    )
 
     if WRITER_RELAY and WRITER_CREDENTIALS:
         creds = get_writer_creds()
-        Sender(SenderConfigSSL(address=(WRITER_RELAY, PORT),
-                               key=creds['key'].name, cert=creds['crt'].name, chain=creds['chain'].name))\
-            .send(tag=HEALTHCHECK_WRITER_TABLE, msg=f'{HEALTHCHECK_WRITER_RECORD}')
+        Sender(
+            SenderConfigSSL(
+                address=(WRITER_RELAY, PORT),
+                key=creds["key"].name,
+                cert=creds["crt"].name,
+                chain=creds["chain"].name,
+            )
+        ).send(tag=HEALTHCHECK_WRITER_TABLE, msg=f"{HEALTHCHECK_WRITER_RECORD}")
 
     if FETCH_INCIDENTS_FILTER:
         alert_filters = check_type(FETCH_INCIDENTS_FILTER, dict)
 
-        assert alert_filters['type'] in [
-            'AND', 'OR'], 'Missing key:"type" or unsupported value in fetch_incidents_filters'
+        assert "type" in alert_filters and alert_filters["type"] in [
+            "AND",
+            "OR",
+        ], 'Missing key:"type" or unsupported value in fetch_incidents_filters'
 
-        filters = check_type(alert_filters['filters'], list)
+        filters = check_type(alert_filters["filters"], list)
 
         for filt in filters:
-            assert filt['key'], 'Missing key: "key" in fetch_incidents_filters.filters configuration'
-            assert filt['operator'] in ['=', '!=', '/=', '>', '<', '>=', '<=', 'and', 'or', '->'], 'Missing key: "operator"'\
-                ' or unsupported operator in fetch_incidents_filters.filters configuration'
-            assert filt['value'], 'Missing key:"value" in fetch_incidents_filters.filters configuration'
-
-    if FETCH_INCIDENTS_DEDUPE:
-        dedupe_conf = check_type(FETCH_INCIDENTS_DEDUPE, dict)
-
-        assert isinstance(dedupe_conf['cooldown'], (int, float)
-                          ), 'Invalid fetch_incidents_deduplication configuration'
+            assert (
+                "key" in filt and filt["key"]
+            ), 'Missing key: "key" in fetch_incidents_filters.filters configuration'
+            assert "operator" in filt and filt["operator"] in [
+                "=",
+                "!=",
+                "/=",
+                ">",
+                "<",
+                ">=",
+                "<=",
+                "and",
+                "or",
+                "->",
+            ], (
+                'Missing key: "operator"'
+                " or unsupported operator in fetch_incidents_filters.filters configuration"
+            )
+            assert (
+                "value" in filt and filt["value"]
+            ), 'Missing key:"value" in fetch_incidents_filters.filters configuration'
 
```

Comment:
since you keep the 'fetch_incidents_deduplication' parameter, please revert these lines:
	
    if FETCH_INCIDENTS_DEDUPE:	
        dedupe_conf = check_type(FETCH_INCIDENTS_DEDUPE, dict)	
        assert isinstance(dedupe_conf["cooldown"], (int, float)	
                          ), "Invalid fetch_incidents_deduplication configuration"

Reply:
I presume this is for those who set it up in previous versions. Am I getting it right? Otherwise, why would we need this code, as this parameter is now hidden and cannot (theoretically) be filled?